modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/TaskTriggerHandler.java

```diff
@@ -51,6 +55,7 @@
 public class TaskTriggerHandler implements TriggerHandler {
 
     private static final String TASK_POSSIBLE_ERRORS_KEY = "task.possible.errors";
+    private static final String SCHEDULER_TASK_TRIGGER_WILDCARD = "org.motechproject.tasks.scheduler(.*)";
```

Comment:
Should be more optimal to use .startsWith() rather than matches for this, since matches uses regex